tests/Application/templates/OrderMessage/Form/_form.html.twig

```diff
@@ -0,0 +1,9 @@
+<h4 class="ui dividing header">{{ 'OrderMessage'|trans }}</h4>
```

Comment:
Remove whole this folder `tests/Application/templates/OrderMessage`. This should bw loaded from your plugin automatically.

Reply:
DONE